scipy/integrate/__quadpack.h

```diff
@@ -27,6 +27,8 @@
 
 #include "numpy/arrayobject.h"
 
+#include "numpy/ndarraytypes.h"
```

Comment:
This shouldn't be needed right, because it's already included through `arrayobject.h`?

Reply:
Sure. I will remove it. I did for safety, since compiler parses a header only once (due to that IFDEF macro thing), so if its already included by some other header, the above line will be ignored. I will try locally removing it and if it works I will push here.